nifi-nar-bundles/nifi-registry-bundle/nifi-registry-service/src/test/java/org/apache/nifi/schemaregistry/services/TestAvroSchemaRegistry.java

```diff
@@ -62,10 +60,10 @@ public void validateSchemaRegistrationFromrDynamicProperties() throws Exception
 
         SchemaIdentifier schemaIdentifier = SchemaIdentifier.builder().name(schemaName).build();
         RecordSchema locatedSchema = delegate.retrieveSchema(schemaIdentifier);
-        assertEquals(fooSchemaText, locatedSchema.getSchemaText().get());
+        Assertions.assertEquals(fooSchemaText, locatedSchema.getSchemaText().get());
         try {
             delegate.retrieveSchema(SchemaIdentifier.builder().name("barSchema").build());
-            Assert.fail("Expected a SchemaNotFoundException to be thrown but it was not");
+            Assertions.fail("Expected a SchemaNotFoundException to be thrown but it was not");
         } catch (final SchemaNotFoundException expected) {
```

Comment:
It looks like this should be changed to use `assertThrows(SchemaNotFoundException.class)`